.env.example

```
@@ -183,4 +183,9 @@ DATA_REPOSITORY=
 TWENTY_CRM_BASE_URL=
 TWENTY_CRM_API_KEY=
 TWENTY_CRM_ENABLED=false
-TWENTY_CRM_SYNC_MODE=disabled
\ No newline at end of file
+TWENTY_CRM_SYNC_MODE=disabled
+
+# 
+NEXT_PUBLIC_SPEED_INSIGHTS_ENABLED=false
+NEXT_PUBLIC_SPEED_INSIGHTS_SAMPLE_RATE=0.5
+NEXT_PUBLIC_VERCEL_ANALYTICS_ID=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)